src/components/Picker/BasePicker/index.js

```diff
@@ -11,6 +11,26 @@ class BasePicker extends React.Component {
         super(props);
 
         this.executeOnCloseAndOnBlur = this.executeOnCloseAndOnBlur.bind(this);
+        this.setDefaultValue = this.setDefaultValue.bind(this);
+    }
+
+    componentDidMount() {
+        this.setDefaultValue();
+    }
+
+    componentDidUpdate(prevProps) {
+        if (prevProps.items === this.props.items) {
```

Comment:
Is this equality check OK or should it use something a little better like `_.isEqual()`?